src/core/evaluator.js

```diff
@@ -1917,7 +1917,10 @@ class PartialEvaluator {
               return;
             }
             // Other marked content types aren't supported yet.
-            args = [args[0].name];
+            args = [
+              args[0].name,
+              args[1] && isDict(args[1]) ? args[1].get("MCID") : null,
```

Comment:
Nit: The following would probably suffice here
```suggestion
              args[1] instanceof Dict ? args[1].get("MCID") : null,
```